Microsoft.Toolkit.HighPerformance/Enumerables/ReadOnlyRefEnumerable{T}.cs

```diff
@@ -116,21 +155,53 @@ internal ReadOnlyRefEnumerable(object? instance, IntPtr offset, int length, int
         {
             this.instance = instance;
             this.offset = offset;
-            this.length = length;
+            Length = length;
             this.step = step;
         }
 #endif
 
+        /// <summary>
+        /// Returns a reference to the first element within the current instance, with no bounds check.
+        /// </summary>
+        /// <returns>A reference to the first element within the current instance.</returns>
+        internal ref readonly T DangerousGetReference()
+        {
+#if SPAN_RUNTIME_SUPPORT
+            return ref MemoryMarshal.GetReference(this.span);
+#else
+            return ref RuntimeHelpers.GetObjectDataAtOffsetOrPointerReference<T>(this.instance, this.offset);
+#endif
+        }
```

Comment:
This method is never used, why did you port it over?

Reply:
If I remember right, I had just implemented both `DangerousGetReference` and `DangerousGetReferenceAt` first and must've neglected to remove `DangerousGetReference` when it wasn't needed.